pkg/odo/cli/application/application.go

```diff
@@ -137,3 +137,25 @@ func getMachineReadableFormat(client *occlient.Client, appName string, projectNa
 	}
 	return appDef
 }
+
+// checkApplicationName checks if the application is present or not
+func checkApplicationName(applicationName string) {
+	if applicationName == "" {
+		printContextError()
+	}
+}
+
+// checkProjectName checks if the project is present in the context or not
+func checkProjectName(context *genericclioptions.Context) {
+	if context.Project == "" {
+		printContextError()
+	}
+}
+
+// printContextError prints a context error if application/project is not found
+func printContextError() {
+	log.Errorf("Please specify the application name and a project name" +
```

Comment:
You don't need the `+` everywhere..

You can format it like so:

```go
log.Errorf(`Please specify the application name and a project name
blahblah
blahblahblah
`)
```

Reply:
But then the `\n` doesn't work in this case and the code looks a bit unformatted.